module1/src/com/epam/linearprograms/Task2.java

```diff
@@ -0,0 +1,15 @@
+package com.epam.linearprograms;
+
+public class Task2 {
+    public static void main(String[] args) {
+        double a = 2.0;
+        double b = 7.0;
+        double c = 2.0;
+        if ((b * b + 4 * a * c >= 0) || a != 0 || b != 0){
+            System.out.println("Value of formula = " + (((b + Math.sqrt(Math.pow(b, 2) +  4 * a * c)) / (2 * a)) - Math.pow(a, 3) * c + 1 / Math.pow(b, 2)));
+        }
+        else if ((b * b + 4 * a * c < 0) || a == 0 || b == 0){
+            System.out.println("Division by 0");
+            }
+        }
```

Comment:
You don't need to check in once again because if thirs condition gives you false, the second condition of course will be true. Moreover if "b * b + 4 * a * c < 0" is true, it's not a division by 0. You can just write that there are incorrect variables.

In addition don't forget about tabulation and spaces